src/Commands/Site/CreateCommand.php

```diff
@@ -26,21 +26,28 @@ class CreateCommand extends SiteCommand implements ContainerAwareInterface
      * @param string $label Site label
      * @param string $upstream_id Upstream name or UUID
      * @option org Organization name, label, or ID
+     * @option region Specify the service region where the site should be
+     *   created, e.g. US or EU. Default is US.
      *
      * @usage <site> <label> <upstream> Creates a new site named <site>, human-readably labeled <label>, using code from <upstream>.
      * @usage <site> <label> <upstream> --org=<org> Creates a new site named <site>, human-readably labeled <label>, using code from <upstream>, associated with <organization>.
      */
 
-    public function create($site_name, $label, $upstream_id, $options = ['org' => null,])
+    public function create($site_name, $label, $upstream_id, $options = ['org' => null, 'region' => null,])
```

Comment:
What is it that the API is going to be doing with an absent region distinction? Can we set the `'region'` to default to 'US' as stated in the comment text?

Reply:
Absence of a region means "us-central1"

Reply:
May we make the default 'us-central1' then? If people will only use it to change it it would be good to make it explicit.

Reply:
Also small bonus: no `isset` check

Reply:
Yes, I agree with you, but see also comment on main thread. We don't yet know if `us-central1` will be exposed to the user or hidden.

Reply:
Deleted comments on account of being too slow.